examples/shadowdom.js

```diff
@@ -0,0 +1,36 @@
+import { check } from 'k6';
+import { browser } from 'k6/x/browser';
```

Comment:
Hmm, but are all our examples with an `x`? I'm not sure what you mean here.

Reply:
We should probably update all our example scripts to import `browser` from `k6/experimental/browser` since we want users to work with the `experimental` import instead of the `x` which signifies an extension. They both work, but we should probably be consistent, like we do in our docs which is to only work with `k6/experimental/browser`.

Happy for the change to be done in a new PR though to fix all the imports.

Reply:
The downside is we can only run the examples with `x` with `xk6`. `x` is useful for rapid development, like changing the local Go code and trying it out against these examples.

Reply:
Right, i see what you mean, otherwise we'd have to build a version of `k6` instead of just being able to do `xk6 run ...` 👍